<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDlg</class>
 <widget class="QDialog" name="SettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCommon">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <widget class="QToolButton" name="tbGpxDir">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tbImportDir">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>80</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leImportDir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>351</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lbGpxDir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>98</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Gpx file directory</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbGpsbabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gpsbabel application</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leGpxDir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>351</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leGpsbabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>351</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="tbGpsbabel">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>130</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbImportDir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Imported track directory</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSigns">
      <attribute name="title">
       <string>Signs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>36</height>
          </size>
         </property>
         <widget class="QToolButton" name="toolButton_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>25</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>25</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_5">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>25</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tvSign"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
